resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -95,8 +95,9 @@ stages:
     stepConditions:
       whitesourceExecuteScan:
         configKeys:
-          - 'product'
+          - 'productName'
```

Comment:
Could you also fix that for the stageDefaults of the security stage below? Actually I copied that part from your PR when I added the defaults for the security stage.

Reply:
Hm. Was already correct. IIRC, I may have changed that in some earlier context.